screensaver.digitalclock/addon.xml

```diff
@@ -0,0 +1,51 @@
+		<description lang="ms_MY">Penyelamat skrin jam digital dengan tarikh, maklumat kini dimainkan, maklumat cuaca, paparan slaid imej dan beberapa pilihan lain.</description>
+		<description lang="pl_PL">Wygaszacz ekranu wyświetlający cyfrowy zegar z datą, informacjami o aktualnie odtwarzanych mediach, o pogodzie, pokaz slajdów i kilkoma innymi opcjami.</description>
+		<description lang="sr_RS">Чувар екрана дигитални сат са датумом, подацима о тренутној репродукцији, подацима о временској прогнози, репродукцијом слајдова слика и неколико опција.</description>
+		<description lang="sr_RS@latin">Čuvar ekrana digitalni sat sa datumom, podacima o trenutnoj reprodukciji, podacima o vremenskoj prognozi, reprodukcijom slajdova slika i nekoliko opcija.</description>
+		<description lang="tr_TR">Dijital saat ekran koruyucu, tarih ve saatin yanı sıra şimdi oynatılıyor bilgisi, hava durumu bilgisi, resim slayt gösterisi gibi çeşitli seçenekler sunar.</description>
+		<disclaimer lang="en_GB">For bugs, requests or general questions visit the Digital clock screensaver thread on the Kodi forum.</disclaimer>
+		<disclaimer lang="en_NZ">For bugs, requests or general questions visit the Digital clock screensaver thread on the Kodi forum.</disclaimer>
+		<disclaimer lang="es_ES">Para errores, peticiones o cuestiones generales visite el hilo Salvapantallas reloj digital en el foro de Kodi</disclaimer>
+		<disclaimer lang="fr_CA">Concernant les bogues ou pour des questions générales, visitez le fil « Digital clock screensaver » du forum de Kodi.</disclaimer>
+		<disclaimer lang="fr_FR">Pour les bogues, demandes ou questions générales, merci de visiter le fil de discussion de l'économiseur d'écran horloge numérique, Digital clock screensaver, sur le forum Kodi.</disclaimer>
+		<disclaimer lang="it_IT">Per bug, richieste o domande generali visita il thread dello Screensaver orologio digitale sul forum Kodi.</disclaimer>
+		<disclaimer lang="lt_LT">Norėdami pranešti apie klaidą, pateikti prašymą ar tiesiog paklausti, aplankykite skaitmeninio laikrodžio ekrano užsklandos temą Kodi forume.</disclaimer>
+		<disclaimer lang="ms_MY">Jika ada pepijat, permintaan atau persoalan umum sila lawati bebenang Digital clock screensaver di dalam forum Kodi.</disclaimer>
+		<disclaimer lang="pl_PL">Odwiedź dedykowany wygaszaczowi wątek na forum Kodi, aby zgłosić błędy, prośbę o funkcje albo zadać ogólne pytanie.</disclaimer>
+		<disclaimer lang="sr_RS">За проблеме, захтеве или општа питања посетите Digital clock screensaver тему на Kodi форуму.</disclaimer>
+		<disclaimer lang="sr_RS@latin">Za probleme, zahteve ili opšta pitanja posetite Digital clock screensaver temu na Kodi forumu.</disclaimer>
+		<disclaimer lang="tr_TR">Hata bildirme, istek ve genel sorularınız için Kodi forumunda dijital saat ekran koruyucu başlığını ziyaret edebilirsiniz.</disclaimer>
+		<platform>all</platform>
+		<license>GNU GENERAL PUBLIC LICENSE. Version 2, June 1991</license>
```

Comment:
Please use the correct SPDX: https://spdx.org/licenses/

Reply:
Hi, I just used what was there originally.
I don't care about licenses nor do I understand them to be honest.

Reply:
I understand, but for Matrix we would like to have a clean start for all add-ons. So replace the license with its SPDX: `GPL-2.0-or-later`

Reply:
So:
<license>GNU GENERAL PUBLIC LICENSE. Version 2, June 1991</license>
Should be:
<license>GPL-2.0-or-later</license>

Reply:
And another question, when I update this according to your instructions, do I need to close this pull request and make a new one or is there a way to merge this change with this pull request?
I know this sounds silly to you but I don't know how to use github, learned it myself so that I can submit this addon to Kodi, so these potentially trivial things are very hard for me.